translations/strings.yaml

```diff
@@ -347,6 +347,7 @@ genericError:
 
 # Connection Info Screen
 connectionInfo:
+  openButton: Connection Info
```

Comment:
Given the use of "info", are there specific limitations on the size of the button that localizers should be mindful of? If so, it would be worth adding it as a comment.